src/quo2/components/messages/system_message.cljs

```diff
@@ -70,7 +70,6 @@
    [rn/view
     {:align-self     :center
      :flex-direction :row
```

Comment:
Is there a reason that the styles are in the same file and not in a separate file?

Reply:
Other components have a structure like this:
```
component-name
 - view.cljs
 - style.cljs
```

Why this one is not like that?